demo/site-configs/site-configs.ts

```diff
@@ -0,0 +1,29 @@
+import { SiteConfig } from "./site-configs.d";
+
+// Called by `npx @comet/cli inject-site-configs`
+export default (): SiteConfig[] => [
+    {
+        name: "Comet Site Main",
+        domains: {
+            main: "localhost:3000",
+        },
+        public: {
+            scope: {
+                domain: "main",
+                languages: ["en", "de"],
+            },
+        },
+    },
```

Comment:
should we also split that into individual files (per domain) to be as similar as possible to starter?

Reply:
Imho we should use in demo the same approach as we have in starter.

And you can start a (independent of this PR) discussion (or create a PR) for changing the approach at any time.